participants/_template.json

```diff
@@ -1,16 +1,36 @@
 {
-	"name": "My Name",
-	"company": "my company",
+	// The schema which is used for validation (see https://json-schema.org/)
+	"$schema": "./_schema.json",
+	// your name / nickname
+	"name": "John Doe",
+	// optional company name
+	"company": "ACME",
+	// either both days or at least one day has to be set to true
 	"when": {
+		// 2023-06-30
 		"friday": true,
+		// 2023-07-01
 		"saturday": false
 	},
+	// your current interests (JS and in general)
 	"tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
+	// if you only eat vegan food
 	"vegan": false,
-	"vegetarian": false,
-	"allergies": "",
-	"what_is_my_connection_to_javascript": "...",
-	"what_can_i_contribute": "???",
-	"tshirt": "/^(W|M)-(S|M|L|XL)$/",
-	"twitter": ""
+	// if you only eat vegan or vegetarian food
+	"vegetarian": true,
+	// what you cannot eat or drink
+	"allergies": ["🥜", "🍏"],
+	// tell us a few words how JavaScript affects you
+	"whatIsMyConnectionToJavascript": "...",
+	// what can you contribute to the bar camp
+	"whatCanIContribute": "???",
+	// if you want a T-Shirt we need your size and variant preference
+	"tShirt": {
+		// S | M | L | XL | XXL
+		"size": "S",
+		// fitted (also known as waist cut or women variant) or regular
+		"type": "fitted"
+	},
+	// your Twitter handle (must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$)
+	"twitter": "yourhandle"
```

Comment:
`mastodon` was mentioned today as an additional account we may want to add

Another idea was adding `"iCanTakeNotesDuringSessions": boolean`, see https://github.com/jscraftcamp/website/issues/1372#issuecomment-1494696984